modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1355,6 +1355,24 @@ definition {
 		Click(locator1 = "Button#SAVE");
 	}
 
+	macro selectAttachmentFileInObjectEntryViaUI {
```

Comment:
How about add the selectAttachmentFileInObjectEntryViaUI in ObjectAutoGenerated.macro file since in which it already has related addAttachment, updateAttachment?